src/libs/Navigation/linkTo.ts

```diff
@@ -182,12 +180,11 @@ export default function linkTo(navigation: NavigationContainerRef<RootStackParam
             action.type = CONST.NAVIGATION.ACTION_TYPE.REPLACE;
 
             // If this action is navigating to the ModalNavigator and the last route on the root navigator is not already opened ModalNavigator then push
```

Comment:
I think we need to adjust this comment as well. And probably can move `action.type = CONST.NAVIGATION.ACTION_TYPE.PUSH;` after the return in the line 199. It's only for Modal navigators